cmd/kn/main_test.go

```diff
@@ -142,9 +142,9 @@ func TestUnknownCommands(t *testing.T) {
 		expectedError []string
 	}{
 		{
-			[]string{"service", "udpate", "test", "--scale-min=0"},
+			[]string{"service", "update", "test", "--scale-min=0"},
 			[]string{"service"},
-			[]string{"unknown sub-command", "udpate"},
+			[]string{"unknown sub-command", "update"},
```

Comment:
this test expects an error given an uknown command, revert this change to make unit tests pass

Reply:
Ah, would `unknown` be acceptable instead?

Reply:
sure